src/virtualization/virtual_machine.rs

```diff
@@ -276,4 +300,8 @@ impl VZVirtualMachine {
             _ => VZVirtualMachineState::Other,
         }
     }
+
+    pub unsafe fn id(&self) -> Id {
+        *self.0
+    }
```

Comment:
Added this for implementing accessing [VZVirtualMachineView setVirtualMachine:] out of this crate. I think the view is not in the scope of this crate.